ggshield/cmd/auth/login.py

```diff
@@ -27,8 +28,21 @@
     type=str,
     help="Name of new token.",
 )
+@click.option(
+    "--lifetime",
+    required=False,
+    type=None,
```

Comment:
You can use `IntRange(0)` for type. This makes Click validate more things for us, and make `--help` output more precise.